CleverTap/Runtime/Native/UnityNativeWrapper/Models/UnityNativeMetadataResponseInterceptor.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using CleverTapSDK.Native;
+using CleverTapSDK.Utilities;
+
+#if (!UNITY_IOS && !UNITY_ANDROID) || UNITY_EDITOR
+namespace Native.UnityNativeWrapper
+{
+    internal class UnityNativeMetadataResponseInterceptor : IUnityNativeResponseInterceptor
```

Comment:
The response interceptor is unused, it is not added to the engine interceptors.

Reply:
Done